### PR TITLE
Adding functionality to /scripts/upgrade_schema.py

### PR DESCRIPTION
**Pull request by Bento007**
- it can now upgrade the published schema and the directory schema.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,6 +32,12 @@ install: docs
 set_oauth2_config:
 	cat ./oauth2_config.json | ./scripts/set_secret.py --secret-name oauth2_config
 
+check_directory_schema:
+	./scripts/upgrade_schema.py
+
+upgrade_directory_schema:
```
Comment (amarjandu) [inline Makefile]: do you want to check before updating?

File: scripts/upgrade_schema.py [hunk 2]
```diff
@@ -15,122 +15,149 @@
...
+parser.add_argument("--stage", required=False, default=os.getenv('FUS_DEPLOYMENT_STAGE'), type=str,
+                    help='The deployment stage to check.')
+parser.add_argument("--upgrade-directory", action="store_true",
+                    help="If set, the directory will be updated to the latest published schema.")
+parser.add_argument("--upgrade-published", action="store_true",
+                    help="If set, the local schema will update the published schema.")
 args = parser.parse_args()
 
+if not args.stage:
+    print("'FUS_DEPLOYMENT_STAGE' not found in environment please run `source environment`, or use the --stage option")
```
Comment (amarjandu) [inline scripts/upgrade_schema.py]: exit(-1) after print?

File: Makefile (hunk 1 above)
Comment (Bento007) [reply Makefile]: Sounds good. Will fix

File: scripts/upgrade_schema.py (hunk 2 above)
Comment (Bento007) [reply scripts/upgrade_schema.py]: will fix

**Pull request merged**